### ISSUE TITLE
Quick Favourites needs a menu entry

### ISSUE THREAD

**Issue by ReinerBforartists** [Bforartists/Bforartists#468]
With hotkey Q you can call your quick favourites menu. Hotkey only functionality is no go.

Task: Quick Favourites needs a menu entry. Maybe we can integrate it into our toolbar editor.

![quickfavourites](https://user-images.githubusercontent.com/13919702/50382018-c5352900-0695-11e9-9fff-1cd4b675222a.jpg)

**Comment by ReinerBforartists** [comment]
Both is better. Have it regular somewhere, and have it optional in the toolbar.

**Comment by ReinerBforartists** [comment]
Hm, something is broken with the hotkey q at the moment. First we need to fix this problem.

**Comment by ReinerBforartists** [comment]
The menu is context and editor type dependant. Doesn't work in topbar or toolbar. We have to add the menu in all editors. Hm, this makes things a bit more complicated ...

![crapsystem](https://user-images.githubusercontent.com/13919702/54350163-0dd50080-464d-11e9-9a71-55500ed478ff.jpg)

**Comment by ReinerBforartists** [comment]
Not really a happy bunny here. I have added it just to the 3D view for now. That's the only location where it could make sense to have. Other editors doesn't have enough functionality to justify this feature.

Problem here is that it will for 99% of the users just collect dust since our menus are not so cluttered like Blender, and we have our toolbar too. Either way, menu functionality needs a menu entry. That's this taks. We can think at a later point about improvements.

Fixed in master.
